framework/logging.cpp

```diff
@@ -636,10 +636,17 @@ void logging_init_global(void)
 
     if (file_log_fd == -1) {
         file_log_fd = real_stdout_fd;
+    } else {
+#ifdef _WIN32
+        // no tty on win32, open app's console instead
+        tty = open("CON:", _O_WRONLY);
```

Comment:
If you redirect the output to a file, do you get the dots in the file?

Reply:
No, dots are only in the console.

Reply:
Thanks.